sdk/log/record.go

```diff
@@ -233,3 +233,20 @@ func (r *Record) Clone() Record {
 	res.back = slices.Clone(r.back)
 	return res
 }
+
+// TestRecord is a log record for testing.
+// You can use it for unit testing [Processor] and [Exporter] implementations.
+// Do not use TestRecord in production code.
+type TestRecord struct {
```

Comment:
How do I get a `Record` from a `TestRecord` so I can use it for testing?

Reply:
`.Record`. it is used in stdoutlog test